presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/CachingPrestoS3FileSystem.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.rubix;
+
+import com.qubole.rubix.core.CachingFileSystem;
+import com.qubole.rubix.spi.ClusterType;
+import io.prestosql.plugin.hive.s3.PrestoS3FileSystem;
+
+public class CachingPrestoS3FileSystem
```

Comment:
There is a whole bunch more classes, see `rubix-prestosql/src/main/java/com/qubole/rubix/prestosql`

Reply:
I should have been more specific in the commit message, though this PR was mainly to verify the CI tests. The moved classes are the ones which reference the Presto classes directly, and thus will break with the package name changes. The othere `CachingXFileSystem` classes only reference Hadoop classes. (I'm not sure why there are so many "cluster manager" related classes but this covers the one that is actually used, at least according to the tests)